pipeline/05-finalize.R

```diff
@@ -64,6 +64,8 @@ metadata <- tibble::tibble(
   assessment_data_year = params$assessment$data_year,
   input_min_sale_year = params$input$min_sale_year,
   input_max_sale_year = params$input$max_sale_year,
+  input_strata_group_var = list(params$input$strata$group_var),
+  input_strata_k_1 = params$input$strata$k_1,
```

Comment:
Saving these variables with the same names as the condo model so they merge into the `metadata` correctly.